editions/tw5.com/tiddlers/community/resources/_Cardo - A GTD-based Task and Project Management Wiki_ by David Szego.tid

```diff
@@ -0,0 +1,19 @@
+created: 20141122093837330
+modified: 20161231232826145
+tags: Resources
+title: "Cardo - A GTD-based Task and Project Management Wiki" by David Szego
```

Comment:
GTD and "Getting things done" are registered trademarks. ... They already forced us once, to rename a TWclassic wiki to GSD (getting stuff done). So my proposal is, to remove the "GTD-based" text all to gether. For me "Task and Project Management" is enough to be found.